tests/spec/core/markdown-spec.js

```diff
@@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\nFoo\n===\n",
+       "\n\nFoo\n===\n",
```

Comment:
I'll post what I'm getting locally tomorrow. 

> On 5 May 2016, at 11:05 PM, Tobie Langel <notifications@github.com> wrote:
> 
> In tests/spec/core/markdown-spec.js:
> 
> > @@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
> >      var ops = {
> >        config: makeBasicConfig(),
> >        body: makeDefaultBody() +
> > -       "\nFoo\n===\n",
> > +       "\n\nFoo\n===\n",
> Not able to reproduce this in http://www.javascriptoo.com/marked
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>